### PR TITLE
Network testing: run tests without exported state

### PR DESCRIPTION
**Pull request by gleb-urvanov**
This branch does not contain exported chainspec as part of CI workflow, nor runtime upgrade.

### CODE REVIEW

**Comment by mnaamani** [comment]
Could you fix the linter errors.

Also when I run yarn its getting modified with:

```
MokoBook:joystream mokhtar$ git diff
diff --git a/yarn.lock b/yarn.lock
index 33280ea44..57a1b8b00 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -3464,7 +3464,7 @@
   dependencies:
     "@babel/runtime" "^7.10.5"
 
-"@polkadot/keyring@^1.7.0-beta.5", "@polkadot/keyring@^3.0.1":
+"@polkadot/keyring@3.0.1", "@polkadot/keyring@^1.7.0-beta.5", "@polkadot/keyring@^3.0.1":
   version "3.0.1"
   resolved "https://registry.yarnpkg.com/@polkadot/keyring/-/keyring-3.0.1.tgz#3944079697c15b2af81e1f57b1c4aeab703a4fef"
   integrity sha512-vAHSBnisiDYHsBbEzAgIpuwQp3vIDN2uWQ/1wAE2BrKzXCBQM7RrF3LRcLFySk0xzQoDs7AP1TlPoakxJ/C/Qw==
```

So perhaps you can update it and check it in also.

File: tests/network-tests/package.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
   "license": "GPL-3.0-only",
   "scripts": {
     "build": "tsc --build tsconfig.json",
-    "test": "tap --files src/iznik/tests/proposals/*Test.ts --files src/iznik/tests/workingGroup/*Test.ts -T",
+    "test": "tap --files src/iznik/tests/unknown.unknown src/iznik/tests/councilSetup.ts src/iznik/tests/proposals/*Test.ts src/iznik/tests/leaderSetup.ts src/iznik/tests/workingGroup/*Test.ts src/iznik/tests/proposals/updateRuntime.ts -T",
```
Comment (mnaamani) [inline tests/network-tests/package.json]: What is `src/iznik/tests/unknown.unknown` ?

Comment (gleb-urvanov) [reply tests/network-tests/package.json]: That is a workaround for `node-tap` `--files` argument issue: the first file in the list will not be processed.

**Comment by mnaamani** [comment]
For sanity check I ran `yarn build` in `tests/network-tests` and I get 76 build errors.
Is that a problem?

**Pull request merged**